packages/fxa-settings/src/pages/ResetPassword/ConfirmResetPassword/index.test.tsx

```diff
@@ -5,39 +5,63 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import ConfirmResetPassword, { viewName } from '.';
-import { usePageViewEvent } from '../../../lib/metrics';
 // import { getFtlBundle, testAllL10n } from 'fxa-react/lib/test-utils';
 // import { FluentBundle } from '@fluent/bundle';
-import { MOCK_EMAIL } from './mocks';
+import { MOCK_EMAIL, MOCK_PASSWORD_FORGOT_TOKEN } from './mocks';
 import { REACT_ENTRYPOINT } from '../../../constants';
+import { LocationProvider, useLocation } from '@reach/router';
+import { Account, AppContext } from '../../../models';
+import { mockAppContext } from '../../../models/mocks';
+import { logPageViewEvent } from '../../../lib/metrics';
 
 jest.mock('../../../lib/metrics', () => ({
   logViewEvent: jest.fn(),
-  usePageViewEvent: jest.fn(),
+  logPageViewEvent: jest.fn(),
 }));
 
+function renderWithAccount(account: Account) {
+  render(
+    <AppContext.Provider value={mockAppContext({ account })}>
+      <LocationProvider>
+        <ConfirmResetPassword />
+      </LocationProvider>
+    </AppContext.Provider>
+  );
+}
+
 const mockNavigate = jest.fn();
 jest.mock('@reach/router', () => ({
   ...jest.requireActual('@reach/router'),
   useNavigate: () => mockNavigate,
+  useLocation: () => {
+    return {
+      state: {
+        email: MOCK_EMAIL,
+        passwordForgotToken: MOCK_PASSWORD_FORGOT_TOKEN,
+      },
+    };
+  },
 }));
 
-describe('ConfirmResetPassword', () => {
+describe.only('ConfirmResetPassword', () => {
```

Comment:
Remove .only?

Reply:
Good eyes!
Looks like **eslint-plugin-jest** has a dedicated [`no-focused-tests`](https://github.com/jest-community/eslint-plugin-jest/blob/HEAD/docs/rules/no-focused-tests.md) rule for this (as well as a really clever [`no-commented-out-tests`](https://github.com/jest-community/eslint-plugin-jest/blob/HEAD/docs/rules/no-commented-out-tests.md) rule).